entrypoint.sh

```diff
@@ -28,6 +28,10 @@ elif [ -n "${DBT_TOKEN}" ]
 then
  echo trying to use DBT_TOKEN/databricks
  sed -i "s/_token_/${DBT_TOKEN}/g" ./datab.yml
+elif [ -n "${INPUT_HTTP_PATH}" ]
```

Comment:
This seems to be wrong: For Databricks, we need to use both, the token and the http path.